aptos-move/framework/aptos-framework/sources/aptos_coin.move

```diff
@@ -44,25 +45,35 @@ module aptos_framework::aptos_coin {
             false, /* monitor_supply */
         );
 
+        (mint_cap, burn_cap)
+    }
+
+    /// Can only be called during genesis for tests to grant mint capability to aptos framework and core resources
+    /// accounts.
+    public(friend) fun configure_accounts_for_test(
```

Comment:
This must not be called for mainnet genesis.

Reply:
If this really important we should probably have a check like `assert!(!running_mainnet(), ...)`.